helm/nats-operator/README.md

```diff
@@ -0,0 +1,139 @@
+# NATS-Operator
+
+NATS is an open-source, cloud-native messaging system. Companies like Apcera, Baidu, Siemens, VMware, HTC, and Ericsson rely on NATS for its highly performant and resilient messaging capabilities.
+
+
+## TL;DR
+
+```bash
+$ helm install .
+```
+
+## Introduction
+
+NATS Operator manages [NATS](https://nats.io/) clusters atop [Kubernetes](http://kubernetes.io), automating their creation and administration. With NATS-operator you can benefits from the flexibility brought by the Kubernetes operator pattern. It means less juggling between manifests and a few handy features like automatic configuration reload.
+
+If you want to manage NATS entirely by yourself and have more control over your NATS cluster, you can always use the original [NATS](https://github.com/helm/charts/tree/master/stable/nats) Helm chart.
+
+
+## Prerequisites
+
+Basic
+- Kubernetes 1.4+ with Beta APIs enabled
+- PV provisioner support in the underlying infrastructure
```

Comment:
sorry not following which Kubernetes feature is this one... we should just make the prerequisites same as in the nats operator readme (`Kubernetes v1.8+`, etc...)

Reply:
I changed it si we only keep `Kubernetes v1.8+`. I don't know what I was thinking. I probably assumed wrongly that NATS and NATS-operator shared the same requirements in terms of Kubernetes configuration. I just pushed a [commit](c3a6cb5417f211920a671d4b718dd3c5ea3d9398) to fix that.